pkg/prove/proof.go

```diff
@@ -16,7 +16,7 @@ import (
 // TxInclusion uses the provided block data to progressively generate rows
 // of a data square, and then using those shares to creates nmt inclusion proofs
 // It is possible that a transaction spans more than one row. In that case, we
-// have to return two proofs.
+// have to return more than one proofs.
```

Comment:
[grammar] honestly may not be worth accepting this suggestion b/c it will dismiss approvals

```suggestion
// have to return more than one proof.
```

Reply:
:grimacing: degen mergingggggggg

Reply:
I will sneak this into a different PR tho lol